Lombiq.HelpfulLibraries.Common/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Linq;
+
+namespace Microsoft.Extensions.DependencyInjection;
+
+/// <summary>
+/// Shortcuts to remove implementations from an <see cref="IServiceCollection"/> instance.
+/// </summary>
+public static class ServiceCollectionExtensions
+{
+    public static IServiceCollection RemoveImplementations<T>(this IServiceCollection services) =>
+        RemoveImplementations(services, typeof(T).FullName);
+
+    public static IServiceCollection RemoveImplementations(this IServiceCollection services, string serviceFullName)
+    {
+        var servicesToRemove = services
+            .Where(service => service.ServiceType?.FullName == serviceFullName)
+            .ToList();
+
+        foreach (var service in servicesToRemove)
+        {
+            services.Remove(service);
+        }

```

Comment:
You could use `List.ForEach()` here:
```suggestion
        servicesToRemove.ForEach(service => services.Remove(service));
```

Reply:
Rather
```suggestion
        servicesToRemove.ForEach(services.Remove);
```

Reply:
Thank you. Now it's beautiful.

Reply:
Does it actually work for you? I get this:

![image](https://user-images.githubusercontent.com/164360/160378863-37b836d0-2e59-41bf-97c2-8d1c18219e5f.png)

Reply:
wth, why does it return `bool`? Never mind then.

Reply:
The return indicates whether the item was found in the collection, which is fine I guess. What I don't understand is why a `Func<T, TOut>` can't be cast to an `Action<TOut>` simply be ignoring the return value 🤔